src/theme/sass-partials/_forms.scss

```diff
@@ -14,30 +15,107 @@ input[type='checkbox'] {
   button#retry {
     float: left;
   }
+
   button#continue {
     float: right;
   }
+
   background-color: map-get($colors, white);
   position: relative;
+
   span {
     font-weight: bold;
   }
+
   signature-pad {
     width: 100%;
     text-align: center;
+
     canvas {
       display: block;
       border: thick solid map-get($colors, divider);
       margin-bottom: 20px;
       width: 100%;
     }
   }
+
   .clearfix {
     clear: both;
   }
 }
 
-.mes-radio-button {
+.mes-checkbox-button {
   height: 20px;
   width: 20px;
 }
+
+.checkbox-label-typography {
+  // todo - add font back later
+  //font-family: 'GDSTransportWebsite';
+  font-size: 19px;
+  letter-spacing: 0.36px;
+}
+
+.checkbox-label {
+  @extend .checkbox-label-typography;
+}
+
+.gds-radio-button {
```

Comment:
Lovely nesting here and handling of element state 😍

Reply:
Many thanks. I hoped to do this with all the text styling and add the GDS font. Tried it but it broke alignment everywhere.